ingestion/src/metadata/ingestion/source/database/sql_column_handler.py

```diff
@@ -211,7 +211,7 @@ def get_columns_and_constraints(
                     )
                     col_data_length = 1 if col_data_length is None else col_data_length
                     om_column = Column(
-                        name=column["name"],
+                        name=column["name"] if column["name"] else " ",
```

Comment:
@OnkarVO7 let's add a comment the reason why we are doing this so we might not clean it in future